examples/esp-todomvc-react/js/TodoItem.jsx

```diff
@@ -0,0 +1,111 @@
+/*global React */
+var app = app || {};
+
+(function () {
+    "use strict";
+
+    var ESCAPE_KEY = 27,
+        ENTER_KEY = 13;
```

Comment:
We use variables here, but in Header.jsx, we just use the number.  Can we consolidate?